draft-ietf-quic-qpack.md

```diff
@@ -359,9 +360,9 @@ infer from this instruction that any updates to the dynamic table have been
 received.
 
 The Header Acknowledgement and Stream Cancellation instructions permit the
-encoder to remove references to entries in the dynamic table.  When entries
-have zero references they are no longer considered blocking (see
-{{blocked-insertion}}).
+encoder to remove references to entries in the dynamic table.  When an entry
+with absolute index lower than the Known Received Count has zero references,
```

Comment:
Does this leave ambiguous what happens to entries with absolute indices higher than Known Received Count?

Reply:
I don't believe so.  It kind of implies that such an entry is considered blocking, which is true.  Note that if the entry is ever referenced in a header block and that reference is acknowledged, that guarantees that the Known Received Count is larger than the absolute index of the entry.